core/src/apps/debug/__init__.py

```diff
@@ -274,18 +279,26 @@ async def dispatch_DebugLinkGetState(
             return DebugLinkState()
```

Comment:
:woman_facepalming: obviously this patch is not helping, you forgot to remove this branch

Reply:
Good catch, thanks!

Reply:
The issue didn't reproduce after an hour of:
```
stress-ng --matrix 12 &
while make -C core test_emu_click TESTOPTS="--durations 0 -k test_repeated_backup"; do :; done 
```